### PR TITLE
Feature/game author classes

### PR DESCRIPTION
**Pull request by saied2035**
## Requirments
- Create a Game class in a separate .rb file.
- Create an Author class associated with the Item class (in a separate .rb file).
- All Game class properties visible in the diagram should be defined and set up in the constructor method.
- All Author class properties visible in the diagram should be defined and set up in the constructor method.

- Implement methods:
     - add_item method in the Author class
          - should take an instance of the Item class as an input
          - should add the input item to the collection of items
          - should add self as a property of the item object (by using the correct setter from the item object)
          
     - can_be_archived?() in the Game class
          - should override the method from the parent class
          - should return true if the parent's method returns true AND if last_played_at is older than 2 years
otherwise, it should return a false

### CODE REVIEW

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @saied2035 
Your project is complete, good job! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 👏 👏 💯
## To highlight 🍾

- All linters are passing ✔️
-  Good commit history ✔️
-  Good PR description ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: author.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require_relative './item'
+# author class
+
+class Author
+  attr_reader :id, :frist_name, :last_name, :items
+
+  def initialize(frist_name, last_name)
+    @id = Random.rand(1..1000)
+    @frist_name = frist_name
```
Comment (jssol) [inline author.rb:9]: There is a typo on `@frist_name = frist_name`
Typos can be a source of bugs in a codebase.
We could change it to `first_name`

Comment (saied2035) [reply author.rb:9]: Hi, @jssol. I will change it before merging. Thanks for you note 😄

**Review by jssol — APPROVED** [review]
Great job implementing these classes!
There is an optional change you can make to improve your code!
Happy coding!

**Pull request merged**